web-ui/src/pages/WorkingLocationPage.jsx

```diff
@@ -9,11 +10,21 @@ const WorkingLocationPage = () => {
       <center>
         <h1>Where would you PREFER to work?</h1>
       </center>
+
+      <div id="workOption">
```

Comment:
This is where the 'centered' text class could go.

Reply:
it has been moved